doc/source/examples/sphinx-gallery/sphinx-gallery.py

```diff
@@ -0,0 +1,96 @@
+# Copyright (C) 2021 - 2024 ANSYS, Inc. and/or its affiliates.
+# SPDX-License-Identifier: MIT
+#
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+# noqa: D205, D400
+"""
+Sphinx gallery
+==============
+
+This example shows how to add a new example to the PyAnsys `Sphinx-Gallery
+<https://sphinx-gallery.github.io/>`_.
+
+To add sphinx-gallery in to your project, you need to install the following packages:
+
+.. code-block:: bash
+
+        pip install sphinx-gallery
+
+then add the following to your ``conf.py`` file:
+
+.. code-block:: python
+
+        extensions = [
+            "sphinx_gallery.gen_gallery",
+        ]
+"""
```

Comment:
```suggestion
# sphinx_gallery_thumbnail_path = '_static/pyansys_light_square.png'

"""
```
Open question on my side. Since we added a thumbnail with `nbsphinx`, why not doing it with `sphinx-gallery` ?
If you agree with that, the proposed change should do the job

Reply:
i added in conf.py file